<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppDetailsPage</class>
 <widget class="QWizardPage" name="AppDetailsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <widget class="QCheckBox" name="autorunCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>128</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Autorun Csound</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>551</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>App Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="appNameLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Target Directory</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="targetDirLineEdit"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
